common/lanelet2_extension/include/lanelet2_extension/regulatory_elements/CarmaTrafficSignal.h

```diff
@@ -139,15 +142,24 @@ class CarmaTrafficSignal : public lanelet::RegulatoryElement
    * @brief prefictState assumes sorted, fixed time, so guaranteed to give you one final state
    *
    * @param time_stamp boost::posix_time::ptime of the event happening
+   * @return std::pair of signal state and its end time at the input time 
```

Comment:
Specify what cases causes the optional<> to be empty

Reply:
Added